packages/tables/resources/views/components/button.blade.php

```diff
@@ -66,6 +67,10 @@
 @if ($tag === 'button')
     <button
         type="{{ $type }}"
+        @if($tooltip)
+        x-data="{ tooltip: {{ json_encode($tooltip) }} }"
```

Comment:
@danharrin What are your thoughts on using the `@js()` directive instead of `json_encode`? It makes debugging things slightly more difficult as Laravel converts more complex structures into base64 encoded strings that get sent through `atob`, but the Blade syntax is a little nicer to look I think...

Reply:
Well, the rendering with @js is better than json_code :
x-data="{ tooltip:'Test'}" vs x-data="{ tooltip:"Test"}"

Reply:
@danharrin Any thoughts ?

Reply:
I finally changed json_encode to @js